logging/logging.go

```diff
@@ -13,7 +13,7 @@ import (
 
 var (
 	globalMu     sync.RWMutex
-	globalLogger = NewDebugLogger("startup")
+	globalLogger = NewDebugLogger("global")
```

Comment:
this logger is used infrequently in various places and not necessarily in startup contexts (i.e. AV code) so I renamed it so that those log lines are a little more clear